django/db/models/base.py

```diff
@@ -878,7 +878,8 @@ def _get_next_or_previous_in_order(self, is_next):
     def prepare_database_save(self, field):
         if self.pk is None:
             raise ValueError("Unsaved model instance %r cannot be used in an ORM query." % self)
-        return getattr(self, field.remote_field.field_name)
+        val = getattr(self, field.remote_field.field_name)
```

Comment:
Okay, my instinct was to use `field.attname`, but I could figure out how to implement it. I can give it another try. Thanks for the feedback.